src/enums/request-code.ts

```diff
@@ -1,9 +1,14 @@
-/** Enum for request codes. */
+/** Enum for Request (action) codes. Not to be confused with NR Action codes. */
 export enum RequestCode {
-  NEW = 'NEW',
-  MOVE = 'MVE',
-  RESTORE = 'REH',
-  AMALGAMATE = 'AML',
-  CHANGE = 'CHG',
-  CONVERT = 'CNV'
+  NEW = 'NEW', // new request
+  MVE = 'MVE', // move request
+  REH = 'REH', // restore or reinstate request
+  AML = 'AML', // amalgamate request
+  CHG = 'CHG', // change name request
+  CNV = 'CNV', // conversion request
+  DBA = 'DBA', // doing business as request
+  ASSUMED = 'ASSUMED', // assumed name request
+  REN = 'REN', // restore with new name request
+  INFO = 'INFO', // special value for sub-menu
+  REST = 'REST', // TODO: possibly typo - same as RESTORE?
```

Comment:
**Does anyone know anything about this? The value was compared in 1 place in code but that's all. Is this a typo?**

Reply:
Was this something we(I) added in at some point? Or is this original code? Hard to say anymore, that being said i don't have any idea.

What was its usage?

Reply:
https://github.com/bcgov/namerequest/blob/d0d3436bbe34565d867929590d34c1d3874c0842/src/store/actions.ts#L486

Reply:
I also noticed that the API uses codes `NRO-NEWAML` and `NRO-REST`. I asked Linda and the team about it. I'll update things as needed...